frontend/src/static/js/main.js

```diff
@@ -247,10 +263,6 @@ window.app = new window.Vue({
       }
     },
 
-    generateKey(dataObj) {
-      return JSON.stringify(dataObj);
```

Comment:
this deletion is due to the method no longer being used

Reply:
Correct.